src/lightning_app/runners/runtime.py

```diff
@@ -51,6 +52,8 @@ def dispatch(
         secrets: Dict of secrets to be passed as environment variables to the app
         cluster_id: the Lightning AI cluster to run the app on. Defaults to managed Lightning AI cloud
         run_app_comment_commands: whether to parse commands from the entrypoint file and execute them before app startup
+        enable_basic_auth: whether to enable basic authentication for the app
+                           (use credentials in the format username:password)
```

Comment:
Why is it called `enable_*` but then the value is a string? Shouldn't it be `basic_auth: str` or `auth: str` or similar?

Reply:
Yeah, we decided to use this now by convention just for simplicity. There is also a restriction to don't have `:` in a password with this format.
CC: @hhsecond